index.md

```diff
@@ -4,8 +4,10 @@
 
 layout: home
 ---
-{:style="text-align: center"}
-![OpTech Logo](/img/logos/optech.png)
+
+<div class="logo">
```

Comment:
I don't have any problem with using HTML here (or anywhere, really), but FYI you can apply a CSS class to a block element in Kramdown like this:

    {:.logo}
    ![OpTech Logo](/img/logos/optech.png)

That will create:

    <p class="logo"><img src="/img/logos/optech.png" alt="OpTech Logo" /></p>

Other things you can do:
- `{#custom-id}`
- `{:custom-tag: "custom-value}`
- Plus other fancier stuff, see the [manual](https://kramdown.gettalong.org/syntax.html#block-ials)

Reply:
Oh, neat! Fixed.